calico-enterprise/operations/ebpf/install.mdx

```diff
@@ -247,6 +247,21 @@ In this example, you would use `60F939227672BC3D5A1B3EC9744B2B21.gr7.us-west-2.e
 </TabItem>
 </Tabs>
 
+### Install the Tigera Operator
+
+Follow the steps in the main install guide for your platform to install the Tigera Operator (and possibly the
+Prometheus Operator). For a KOPS cluster, patch the tigera-operator deployment with dns config, so that the operator
+can resolve the apiserver DNS.
+
+```bash
+kubectl patch deployment -n tigera-operator tigera-operator -p '{"spec":{"template":{"spec":{"dnsConfig":{"nameservers":["127.0.0.53"]}}}}}'
+```
+
+However, **stop** before applying the `custom-resources.yaml`; we'll customise that file
```

Comment:
I'm afraid folks might read the first sentence and miss the note about not applying the yaml. How about this:

Follow the steps in the main install for your platform that installs the Tigera Operator, **without applying the `custom-resources.yaml`** (you will update this file in a later step in this doc).  If you have a KOPS cluster, you must also patch the tigera-operator deployment with DNS config so the operator can resolve the apiserver DNS.